### ISSUE TITLE
Cannot install 2019.4.4f1

### ISSUE THREAD

**Issue by brennenawana** [DragonBox/u3d#401]
### Issue Checklist

- [x] I'm using the latest u3d version
- [x] I ran `u3d --help`
- [x] I read the [README](https://github.com/DragonBox/u3d/blob/master/README.md)
- [x] I made sure that a similar [issue](https://github.com/DragonBox/u3d/issues) doesn't exit

### Issue Description

I am unable to install Version 2019.4.4f1
I run `u3d install 2019.4.4f1 -p Unity,Android,iOS` and expected it to install but instead see
```
Fetching central 'versions.json' cache
No version '2019.4.4f1' was found in cache. Either it doesn't exist or u3d doesn't know about it yet.
Try refreshing with u3d available -f
```
I run `u3d availalbe -f` and expect it to be in the list, but it's not.

**Comment by niezbop** [comment]
This is a side effect of #402 

Discussion regarding this issue will be discussed there.

**Comment by niezbop** [comment]
Fixed now:

```
$ u3d available --no-central --force -u 2019.4.4f1
Cache is out of date. Updating cache for Mac OSX
Loading Unity lts releases
Found 71 lts releases.
Loading Unity stable releases
Found 109 stable releases.
Loading Unity patch releases
Found 0 patch releases.
Loading Unity beta releases
Found 0 beta releases.
Version 2019.4.4f1: https://download.unity3d.com/download_unity/1f1dac67805b/
```

The central cache will be updated sometimes in the coming days, and will have 2019.4.4f1 then.
